content/nginxaas-azure/loadbalancer-kubernetes.md

```diff
@@ -2,7 +2,7 @@
 title: NGINXaaS Load Balancer for Kubernetes
 weight: 250
 toc: true
-url: /nginxaas/azure/quickstart/loadbalancer-kubernetes/
+url: /nginxaas/azure/loadbalancer-kubernetes/
```

Comment:
Has this redirect been tested? I know we had issues with them not working in the past, but I can't remember if that was back on the Netlify site or not.

Reply:
addressed in slack

Reply:
Ahh, nm, I was thinking of `alias`